.circleci/config.yml

```diff
@@ -18,7 +18,8 @@ commands:
           command: |
             virtualenv env -q
             source env/bin/activate
-            pip install . -q
+            pip install . --progress-bar off
```

Comment:
Running 
```
pip config --site set global.progress_bar off
```
Makes the progress bar off by default

If we add that line above this one, inside the installation command, it should remove the need to use the progress bar flag in any of the pip install commands in this file.